docs/rules/has-valid-tappable-area-size.md

```diff
@@ -0,0 +1,59 @@
+# has-valid-tappable-area-size
+
+Enforce that <Touchable\*> components comply with the minimum [Target Size Area specified in the WCAG](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html).
+
+In case that porcentual values are being used to determine the height and width, even if they're higher than 44px still the linter will demand to explicitly specify the minHeight and minWidth.
```

Comment:
```suggestion
The way the rule works is by requiring a `minWidth` and `minHeight` for the <Touchable /> components.

In case that porcentual values are being used to determine the height and width, even if they're higher than 44 points, the linter will still enforce that the `minHeight` and `minWidth` are explicitly specified.

For that reason and to not have to apply these properties to your touchables each time. We recommend having a base touchable component that has these style rules.
```